scss/imports/_header.scss

```diff
@@ -0,0 +1,23 @@
+// dat-header
+
+.dat-header {
+  height: 3rem;
+  padding: .5rem .75rem;
+  background-color: $color-neutral;
+  color: $color-white;
+  -webkit-app-region: drag;
+}
+
+.menu-trigger {
+  height: 2rem;
+  padding-left: .625rem;
+  width: 2.5rem;
+  .icon-menu {
+    fill: $color-neutral--20;
+  }
+  &:hover, &:focus {
+    .icon-menu {
```

Comment:
Yup, that's understood correctly. I love the `&` selector for this particular use case: I almost always use the same styles for `&:hover, &:focus` for easy accessibility wins and find the grouping helpful – but can see that it’s a bit hard to read if not used to it.
Can we have `&` in sheetify at all?